linode/nodepool/framework_resource_schema.go

```diff
@@ -0,0 +1,74 @@
+package nodepool
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework-validators/int64validator"
+	"github.com/hashicorp/terraform-plugin-framework-validators/listvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/linode/terraform-provider-linode/v2/linode/helper"
+)
+
+var resourceSchema = schema.Schema{
+	Version: 1,
```

Comment:
Since this is the first version of this schema, this `Version` attribute can be dropped and the version will default to `0` 👍

Reply:
done